cli/ast/bundle_hook.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
```

Comment:
This file is a straight extract out of `mod.ts`. I did a slight bit of cleanup in this PR as well.